docs/hugo/content/getting-started/_index.md

```diff
@@ -0,0 +1,367 @@
+---
+title: Getting Started
+weight: 5
+type: docs
+description: "Install and get started with Azure Service Operator"
+---
+
+## Prerequisites
+
+1. A Kubernetes cluster (at least version 1.16) [created and running](https://kubernetes.io/docs/tutorials/kubernetes-basics/create-cluster/). You can check your cluster version with `kubectl version`. If you want to try it out quickly, spin up a local cluster using [Kind](https://kind.sigs.k8s.io).
+2. An Azure Subscription to provision resources into.
+3. An Azure Service Principal for the operator to use, or the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest) to create one. How to create a Service Principal is covered in [installation](#installation).
+   See the [Azure Workload Identity]( {{< relref "credential-format#managed-identity-via-workload-identity" >}} ) setup for how to use managed identity instead. We recommend using workload identity in production.
+
+## Installation
+
+### Install cert-manager on the cluster
```

Comment:
This reminds me that we may have a problem with a dependency on an old version of cert-manager. I'll bump this for tomorrow's weekly sync.